src/openrct2/peep/Guest.cpp

```diff
@@ -6320,7 +6320,7 @@ static void peep_update_walking_break_scenery(Peep* peep)
 
     tileElement->AsPath()->SetIsBroken(true);
 
-    map_invalidate_tile_zoom1(peep->next_x, peep->next_y, (tileElement->GetBaseZ()) + 32, tileElement->GetBaseZ());
+    map_invalidate_tile_zoom1({ peep->next_x, peep->next_y, (tileElement->GetBaseZ()) + 32, tileElement->GetBaseZ() });
```

Comment:
remove the additional brackets around GetBaseZ()

Reply:
This is also very odd why is the base higher than the clearance.

Reply:
Just checked the assembly this is the wrong way round. should be `tileElement->GetBaseZ(), tileElement->GetBaseZ() + 32`

Reply:
Nice catch, just fixed it!

Reply:
Nice catch indeed - would this fix reduce the number of Z fighting issues?